examples/pie_and_polar_charts/nested_pie.py

```diff
@@ -36,28 +49,29 @@
 # the exact design of the plot.
 #
 # In this case, we need to map x-values of the bar chart onto radians of
-# a circle.
+# a circle. The cumulative sum of the values are used as the edges
+# of the bars.
 
 fig, ax = plt.subplots(subplot_kw=dict(polar=True))
 
-left_inner = np.arange(0.0, 2 * np.pi, 2 * np.pi / 6)
-left_middle = np.arange(0.0, 2 * np.pi, 2 * np.pi / 12)
-left_outer = np.arange(0.0, 2 * np.pi, 2 * np.pi / 9)
+size = 0.3
+vals = np.array([[60., 32.], [37., 40.], [29., 10.]])
+#normalize vals to 2 pi
+valsnorm = vals/np.sum(vals)*2*np.pi
+#obtain the ordinates of the bar edges
+valsleft = np.cumsum(np.append(0, valsnorm.flatten()[:-1])).reshape(vals.shape)
 
-ax.bar(x=left_inner,
-       width=2 * np.pi / 6, bottom=0, color='C0',
-       linewidth=2, edgecolor='w',
-       height=np.zeros_like(left_inner) + 5)
+cm = plt.get_cmap("tab20c")
+cout = cm(np.arange(3)*4)
+cin = cm(np.array([1, 2, 5, 6, 9, 10]))
 
-ax.bar(x=left_middle,
-       width=2 * np.pi / 12, bottom=5, color='C1',
-       linewidth=2, edgecolor='w',
-       height=np.zeros_like(left_middle) + 2)
+ax.bar(x=valsleft[:, 0],
+       width=valsnorm.sum(axis=1), bottom=1-size, height=size,
+       color=cout, edgecolor='w', linewidth=1, align="edge")
 
-ax.bar(x=left_outer,
-       width=2 * np.pi / 9, bottom=7, color='C2',
-       linewidth=2, edgecolor='w',
-       height=np.zeros_like(left_outer) + 3)
+ax.bar(x=valsleft.flatten(),
+       width=valsnorm.flatten(), bottom=1-2*size, height=size,
+       color=cin, edgecolor='w', linewidth=1, align="edge")
 
 ax.set(title="Pie plot with `ax.bar` and polar coordinates")
```

Comment:
Out of curiosity: Do we have general recommendations for `ax.set(prop=...)` vs. `ax.set_prop(...)`? I'm using the latter one for single properties.

Reply:
fwiw I really like `.set(foo=bar)` for everything.